applications/serial_lte_modem/doc/PPP_AT_commands.rst

```diff
@@ -93,3 +95,32 @@ Example
   #XPPP: 0,0
 
   OK
+
+Testing on Linux
+================
+
+You can test SLM's PPP on Linux by using the ``pppd`` command.
+
+For that, SLM must be compiled without CMUX because there is no widely available utility that allows multiplexing a device file on Linux.
+If you happen to be equipped with one, you can use SLM's PPP with ``pppd`` through CMUX by first setting up CMUX and replacing the device file in the ``pppd`` command by that of SLM's PPP channel (that will have been created by the CMUX utility).
+
+Assuming that SLM's UARTs are connected to the Linux host, you first need to get PPP running.
+For that, start SLM and issue an ``AT+CFUN=1`` command.
```

Comment:
```suggestion
You first need to get PPP running.
For that, start SLM and issue an ``AT+CFUN=1`` command.
```
I moved the first part higher.

Can you format these as numbered steps, preferably so that a step isn't just "Run the following command", but rather either "Run the following command to [...]" or "[Do something]. To do this, run the following command." so the reader knows what they are doing when they run the commands.